src/LibraryManager.Vsix/Json/Completion/CompletionController.cs

```diff
@@ -123,7 +123,7 @@ private void OnCommitted(object sender, EventArgs e)
             }
 
             var doc = JSONEditorDocument.FromTextBuffer(_textView.TextDataModel.DocumentBuffer);
-            JSONParseItem parseItem = doc?.JSONDocument.ItemBeforePosition(_textView.Caret.Position.BufferPosition);
```

Comment:
Not sure if it could be null. The only instance I can think of is if the document was closed soon after completion session is invoked. I added a null check: https://github.com/aspnet/LibraryManager/commit/a7836411bef714a2f88fe66ca4c31ad2b2a155ae